src/pageServices/contentScripts/highlight.js

```diff
@@ -128,10 +128,17 @@ export const highlightOnPage = () => {
     div.querySelector(".jdn-class").textContent = element.name;
   };
 
+  const updateTooltipXpath = (element) => {
+    if (element.locator.taskStatus === "SUCCESS" && tooltip.getAttribute("jdn-element-id") === element.element_id) {
```

Comment:
We have `LocatorTaskStatus` enum in `src/features/locators/types/locator.type.ts` and you can use it here for check

Reply:
Thank you for the tip! Updated 😊